test/common/crudtester.go

```diff
@@ -79,17 +79,15 @@ func NewFederatedTypeCrudTester(testLogger TestLogger, typeConfig typeconfig.Int
 	}, nil
 }
 
-func (c *FederatedTypeCrudTester) CheckLifecycle(desiredFedObject *unstructured.Unstructured) {
+func (c *FederatedTypeCrudTester) CheckLifecycle(desiredFedObject *unstructured.Unstructured, orphanDependents *bool) {
```

Comment:
`CheckLifecycle` is an expensive test, which is why it only validates the golden path. That doesn't suggest adding `orphanDependents` parameter to validate the non-golden path.